src/code.sh

```
@@ -86,7 +86,12 @@ main() {
     if [ "$add_samplename_column" == true ]; then args+="--add_name "; fi
     if [ "$add_comment_column" == true ]; then args+="--add_comment_column "; fi
     if [ "$add_classification_column" == true ]; then args+="--add_classification_column "; fi
+    if [ "$add_allele_origin_column" == true ]; then args+="--add_allele_origin_column "; fi
```

Shellcheck (0.10.0)

[warning] add_allele_origin_column is referenced but not assigned.

(SC2154)